# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsSocketTransportService2.h
# Commit: f8d31e3ebc93
# Full Hash: f8d31e3ebc93db2a466859043fddd66459ac9b04
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2021-09-22 15:53:06
# Regressor Bug: 1382886
# File Overlap Count: 1
# Description:
#   Bug 1382886 - Use HandshakeDone to signal that a handshake is done r=necko-reviewers,keeler,kershaw
#   
#   HandshakeDone will be called after a handshake is finished and also after the certificate verifications are done.
#   The code relies on HandshakeDone to signal that the handshake is done. When early-data is not available HandshakeDone is responsible for setting up a Http2 session if needed. There are 2 outcomes when early-data is used:
#   1) early-data is accepted and transaction continues polling for read,
# ==============================================================================

diff -r 971bda7b5986 -r f8d31e3ebc93 netwerk/base/nsSocketTransportService2.h
--- a/netwerk/base/nsSocketTransportService2.h	Wed Sep 22 09:19:30 2021 +0000
+++ b/netwerk/base/nsSocketTransportService2.h	Wed Sep 22 09:19:31 2021 +0000
@@ -126,8 +126,6 @@
   bool IsTelemetryEnabledAndNotSleepPhase();
   PRIntervalTime MaxTimeForPrClosePref() { return mMaxTimeForPrClosePref; }
 
-  void SetNotTrustedMitmDetected() { mNotTrustedMitmDetected = true; }
-
   // According the preference value of `network.socket.forcePort` this method
   // possibly remaps the port number passed as the arg.
   void ApplyPortRemap(uint16_t* aPort);
@@ -344,8 +342,6 @@
 
   void TryRepairPollableEvent();
 
-  bool mNotTrustedMitmDetected{false};
-
   CopyableTArray<nsCOMPtr<nsISTSShutdownObserver>> mShutdownObservers;
 };
 